atoma-proxy/src/server/streamer.rs

```diff
@@ -418,7 +412,7 @@ impl Stream for Streamer {
                         "Error verifying response: {}",
                         e
                     );
-                    Error::new(format!("Error verifying and signing response: {}", e))
+                    Error::new(format!("Error verifying and signing response: {e:?}"))
```

Comment:
Do you need the `:?` ?

Reply:
Yes see https://github.com/atoma-network/atoma-proxy/pull/175#discussion_r1936051622